src/base_model.jl

```diff
@@ -305,12 +460,12 @@ function build_base_model!(ECModel::AbstractEC, optimizer; use_notations=false)
 
 
     # Set the minimum level of the energy stored in the battery to be proportional to the capacity
-    @constraint(model_user, con_us_min_E_batt[u in user_set, b in asset_names(users_data[u], BATT), t in time_set],
+    @constraint(model_user, con_us_min_E_batt[u in user_set, b in asset_names(users_data[u], THER), t in time_set],
         x_us[u, b] * field_component(users_data[u], b, "min_SOC") - E_batt_us[u, b, t] <= 0
     )
 
     # Set the maximum level of the energy stored in the battery to be proportional to the capacity
-    @constraint(model_user, con_us_max_E_batt[u in user_set, b in asset_names(users_data[u], BATT), t in time_set],
+    @constraint(model_user, con_us_max_E_batt[u in user_set, b in asset_names(users_data[u], THER), t in time_set],
         - x_us[u, b] * field_component(users_data[u], b, "max_SOC") + E_batt_us[u, b, t] <= 0
     )

```

Comment:
Please, restore these changes; we need a new block of lines for STOR/TES or alike.